doc/radosgw/placement.rst

```diff
@@ -123,6 +124,18 @@ Then provide the zone placement info for that target:
         --index-pool default.rgw.temporary.index \
         --data-extra-pool default.rgw.temporary.non-ec
 
+.. note:: With default placement target settings, RGW stores an object's first data chunk in the RADOS "head" object along
+          with xattr metadata. The `--placement-inline-data=false` flag may be passed with the `zone placement add` or
+          `zone placement modify` commands to change this behavior for new objects stored on the target.
+          When data is stored inline (default), it may provide an advantage for read/write workloads since the first chunk of
+          an object's data can be retrieved/stored in a single librados call along with object metadata. On the other hand, a
+          target that does not store data inline can provide a performance benefit for RGW client delete requests when
+          bluestore db is located on faster storage devices (as compared to data devices) since it eliminates the need to access
+          slower devices synchronously while processing the client request. In that case, all data associated with the deleted
+          objects can be removed asynchronously in the background by garbage collection.                                          
```

Comment:
we don't have any experience with this feature yet, so i think the documentation for these backports should note that it's experimental and not officially supported

Reply:
added a note at the end of the blurb